packages/cf-component-checkbox/src/CheckboxGroup.js

```diff
@@ -27,7 +38,7 @@ class CheckboxGroup extends React.Component {
 
   render() {
     return (
-      <div className="cf-checkbox__group">
+      <div>
         {this.props.options.map(option => {
```

Comment:
Yeah I think I'd like to try to get stuff to `fela` first and then look at stuff like this. Maybe we could have a codebase review or something to look over the codebase and clean some of the stuff like this up.